decidim-core/spec/system/notifications_spec.rb

```diff
@@ -88,12 +88,12 @@
     end
 
     context "when setting a single notification as read" do
-      let(:notification_title) { "An event occured to #{translated resource.title}" }
+      let(:notification_title) { "An event occured to #{decidim_sanitize_translated resource.title}" }
```

Comment:
Let's keep it as it was in develop 🙏🏽 
```suggestion
      let(:notification_title) { "An event occured to #{translated resource.title}" }
```